egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -50,6 +50,9 @@ def GetArgs():
                         default=0.0)
     parser.add_argument("--include-log-softmax", type=str, action=nnet3_train_lib.StrToBoolAction,
                         help="add the final softmax layer ", default=True, choices = ["false", "true"])
+    parser.add_argument("--ivector-interval", type=int,
+                        help="If positive, Round descriptor will be used for ivector instead of ReplaceIndex descriptor, and the value will be the value of <t-modulus>.", default=10)
```

Comment:
I think better to describe it in a less technical way, i terms of the period with which the iVectors are supplied.  And split the string on multiple lines.